fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -51,12 +51,15 @@ def simulators(requested_os_type = "")
       def connected_devices(requested_os_type)
         UI.verbose("Fetching available connected devices")
 
+        tvos_devices = ["AppleTV"]
+        ios_devices = ["iPhone", "iPad", "iPod"]
+
         device_types = if requested_os_type == "tvOS"
-                         ["AppleTV"]
+                         tvos_devices
                        elsif requested_os_type == "iOS"
-                         ["iPhone", "iPad", "iPod"]
+                         ios_devices
                        else
-                         []
+                         ios_devices + tvos_devices
```

Comment:
Hey @mpirri this change was needed due to a upstream change in scan that stopped providing the device type in some cases. This would result in usb device detection being run but filtering out all results and prevented the configuration i was using from discovering devices.